Formula/g/gbdfed.rb

```diff
@@ -57,4 +57,4 @@ def install
 +	-DGDK_PIXBUF_DISABLE_DEPRECATED
  
  SRCS = bdf.c \
-        bdfcons.c \
+        bdfcons.c
```

Comment:
This is not Ruby code and sounds like a bug in RuboCop

Reply:
Thanks! Happy to merge the `portaudio` and `util-linux` changes in the meantime and we can wait for 1.68.1 or something.